contracts/optimistic-ethereum/mockOVM/bridge/mockOVM_CrossDomainMessenger.sol

```diff
@@ -29,6 +29,7 @@ contract mockOVM_CrossDomainMessenger is OVM_BaseCrossDomainMessenger {
      **********************/
 
     ReceivedMessage[] internal fullReceivedMessages;
+    address public targetMessengerAddress;
```

Comment:
Let's make this internal, since it's no longer in the real CrossDomainMessengers